services/simcore/docker-compose.yml.j2

```diff
@@ -189,7 +189,7 @@ services:
         - traefik.http.routers.${SWARM_STACK_NAME}_invitations.rule=(${DEPLOYMENT_FQDNS_CAPTURE_INVITATIONS})
         - traefik.http.routers.${SWARM_STACK_NAME}_invitations.entrypoints=http
         - traefik.http.services.${SWARM_STACK_NAME}_invitations.loadbalancer.server.port=8000
-        - traefik.http.routers.${SWARM_STACK_NAME}_invitations_swagger.rule=(${DEPLOYMENT_FQDNS_CAPTURE_INVITATIONS}) && PathPrefix(`/dev/doc`)
+        - traefik.http.routers.${SWARM_STACK_NAME}_invitations_swagger.rule=(${DEPLOYMENT_FQDNS_CAPTURE_INVITATIONS}) && (PathPrefix(`/dev/doc`) || PathPrefix(`/doc/`))
```

Comment:
not here. The [issue](https://github.com/ITISFoundation/osparc-ops-environments/issues/1033) never mentioned this  :-)

Reply:
fixed :--)